docs/guides/options.md

```diff
@@ -1,138 +1,352 @@
+### Precedence
 
+There are multiple ways to provide options for Video.js, so we should be clear about which methods take precedence. From highest to lowest, the precedence rules are:
 
-### controls ###
-The controls option sets whether or not the player has controls that the user can interact with. Without controls the only way to start the video playing is with the autoplay attribute or through the API.
+1. Options as the second argument to the `videojs` function
+1. Options parsed from the `data-setup` attribute
+1. `<video>` element attributes and children
+
+For example, given:
 
 ```html
-<video controls ...>
-or
-{ "controls": true }
+<video id="foo" src="foo-1.mp4" data-setup='{"src":"foo-2.mp4"}'></video>
+<script>
+  videojs('foo', {src: 'foo-3.mp4'});
+</script>
 ```
 
+The `src` option would have the value `'foo-3.mp4'`.
 
-### autoplay ###
-If autoplay is true, the video will start playing as soon as page is loaded (without any interaction from the user).
-NOT SUPPORTED BY APPLE iOS DEVICES. Apple blocks the autoplay functionality in an effort to protect its customers from unwillingly using a lot of their (often expensive) monthly data plans. A user touch/click is required to start the video in this case.
-```html
-<video autoplay ...>
-or
-{ "autoplay": true }
-```
+## Options Reference
 
+### Standard `<video>` Element Options
 
-### preload ###
-The preload attribute informs the browser whether or not the video data should begin downloading as soon as the video tag is loaded. The options are auto, metadata, and none.
+Each of these options is also available as a standard `<video>` element attribute; so, they can be defined in all three manners [outlined above](#setting-options). Typically, defaults are not listed as this is left to browser vendors.
 
-'auto': Start loading the video immediately (if the browser agrees). Some mobile devices like iPhones and iPads will not preload the video in order to protect their users' bandwidth. This is why the value is called 'auto' and not something more final like 'true'.
+#### `autoplay`
 
-'metadata': Load only the meta data of the video, which includes information like the duration and dimensions of the video.
+> Type: `Boolean`
 
-'none': Don't preload any of the video data. This will wait until the user clicks play to begin downloading.
+If `true`/present as an attribute, begins playback automatically.
 
-```html
-<video preload ...>
-or
-{ "preload": "auto" }
-```
+> **Note:** As of iOS 10, Apple offers `autoplay` support in Safari. However, in order for it to work, the `<video>` element must have `muted` and `playsinline` attributes!
```

Comment:
If it doesn't have audio tracks it also works.
We should also double-check whether `playsinline` is required.